subsys/fs/littlefs_fs.c

```diff
@@ -357,6 +360,7 @@ static off_t littlefs_tell(struct fs_file_t *fp)
 	return ret;
 }
 
+#ifndef LFS_READONLY
```

Comment:
This `#ifndef LFS_READONLY` is within block that is already included via `#ifndef LFS_READONLY`.